### ISSUE TITLE
Xamarin Authentication Webview Hide Cancel Button on Login

### ISSUE THREAD

**Issue by aherrick** [AzureAD/azure-activedirectory-library-for-dotnet#902]
Is it possible via the iOS/Android/UWP implementations to remove the "Cancel" button?

I don't want my users to be able to even cancel out of the authentication as the entire App relies on an Authenticated user.

**Comment by jmprieur** [comment]
I'm not sure that I understand your question completely. Which cancel button is it?
I'm assuming that you are trying to sign-in and you get a page where you need to enter your email address, and there is a cancel button ? if this is the cancel button you mean, then, no there is no way to hid it. It's part of a web page provided by the Azure AD service for the authentication.

**Comment by aherrick** [comment]
Yes I am talking about the web view that prompts when trying to login on Xamarin Mobile (iOS/Android)

Are you saying that wouldn't be a feature that could be added? I think it's a valid request IMO. Is this not the correct repository to ask in?

Thanks!

**Comment by jmprieur** [comment]
Thanks for the feedback @aherrick .
What you propose is a good idea, which would be surfaced in the auth libraries, but really require changes on the service.
I suggest you post a suggestion on uservoice: https://feedback.azure.com/forums/169401-azure-active-directory?category_id=164757
